examples/support/client.rs

```diff
@@ -98,12 +102,23 @@ impl Client {
 
         let mut params = HashMap::new();
         params.insert("grant_type", "authorization_code");
-        params.insert("client_id", &self.config.client_id);
         params.insert("code", code);
         params.insert("redirect_uri", &self.config.redirect_uri);
-        let access_token_request = client
-            .post(&self.config.token_url)
-            .form(&params).build().unwrap();
+        let access_token_request =  match &self.config.client_secret{
```

Comment:
I add param client_secret so users can test both public and private clients.  When it's a private client, add base64 authheader to the request.

Reply:
That's a good idea in general, thank you.